index.html

```diff
@@ -0,0 +1,215 @@
+<!DOCTYPE html>
+<html lang="pl">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Zadanie 5</title>
+    <link rel="stylesheet" href="./styles/base.css">
+    <link rel="stylesheet" href="./styles/header.css">
+    <link rel="stylesheet" href="./styles/content.css">
+    <link rel="stylesheet" href="./styles/footer.css">
+</head>
+
+<body>
+
+    <!-- Pytanie czy w VSC moge tworzyć własne szablony i donieśc się do nich jak w Emmet? -->
+
+    <!-- Header start -->
+
+    <header class="header row">
+        <div class="header__container container">
+            <h1 class="header__logo cell">
+                <a href="#"><img src="images/logo.svg" alt=""></a>
+            </h1>
+            <nav class="header__nav cell">
+                <ul>
+                    <li><a href="#">Team</a></li>
+                    <li><a href="#">Pricing</a></li>
+                    <li><a href="#">Features</a></li>
+                </ul>
+            </nav>
+        </div>
+    </header>
+
+    <!-- Slider -->
+
+    <section data-id="Slider" class="content row">
+        <div class="content__container container">
+            <article class="content__article cell left">
+                <title>Beautiful Free Nova template</title>
+                <p>A top notch premium quality template for your next web project.</p>
+                <button type="button">Download FREE!</button>
+            </article>
+            <article class="content__article cell right">
```

Comment:
Na pewno nie robiłbym 2 razy `article`. Bo ten tag dla samego `img` to za mało.
Zrobiłbym odwrotnie tj. `div` zamieniłbym na `article`, a `div` dodał do treści artykułu, aby móc go wypozycjonować. Natomiast zamiast okalać `img` przez `article`, to zrobiłbym `figure`.

PS. Tak `title` jest wykorzystywany jedynie w `head`. Tutaj powinno być `h1` lub inny poziom